lib/local_authorities_importer.rb

```diff
@@ -0,0 +1,63 @@
+require 'ostruct'
+
+class LocalAuthoritiesImporter
+  include JsonDownloader
+
+  LOCAL_AUTHORITY_TYPES = "DIS,LBO,MTD,UTA,COI,LGD"
+
+  def self.import_from_mapit
+    new.authorities_from_mapit
+  end
+
+  def authorities_from_mapit
+    url = "#{MAPIT_BASE_URL}areas/#{LOCAL_AUTHORITY_TYPES}"
+    mapit_las = get_mapit_authorities(url)
+
+    mapit_las.each do |mapit_la|
```

Comment:
I'm now using `gds-api-adapters` after @jennyd 's suggestion so we get around that.